compiler/onnx-subgraph/Readme.md

```diff
@@ -0,0 +1,48 @@
+## onnx_autosubgraphs
+in this project, we can support onnx AI model auto subgraph, the AI model can be  splitted by model size, operators etc, it help much for on-device AI acceleration, and has been verified on Rose-P NPU and Qualcomm DSP.
+
+### OS environment
+     1. ubuntu >=20.04
+     2. GCC >= 9.4.0
+     3. cmake >= 3.10
+     4. python >= 3.8
+     5. apt-get install libprotobuf-dev protobuf-compiler
+
+### Python packages
+    onnx                         1.16.0
+    onnxruntime                  1.18.1
+    onnxsim                      0.4.36
+    torch                        2.3.1
+
+### Pre-steps
+    1. prepare the target onnx AI model, we use test.onnx for example
+    2. use onnxsim to remove the complex structures before excution onnx-subgraph
+    
+### building the onnx-subgraph
+    1. cd onnx-subgraph
+    2. mkdir build & cd build
+    3. cmake .. & make
```

Comment:
we don't allow building separate building in `compiler/*` modules
all compiler modules are built from `infra/nncc/CMakeLists.txt`
have you not read `docs/howto/how-to-build-compiler.md` ?